Backend/dev/test-tool/context-api/server.py

```
@@ -3998,6 +4067,172 @@ def do_DELETE(self):
+    payload = "\n".join(lines) + "\n"
+    try:
+        req = urllib.request.Request(
+            VICTORIA_METRICS_URL,
+            data=payload.encode("utf-8"),
+            headers={"Content-Type": "text/plain"},
+            method="POST",
+        )
+        with urllib.request.urlopen(req, timeout=10) as r:
+            r.read()
+        return True, f"OK — {len(lines)} metrics sent to {VICTORIA_METRICS_URL}"
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)